src/hooks/useOperatorTrades.ts

```diff
@@ -97,8 +97,10 @@ export function useOrderTrades(order: Order | null): Result {
 
     fetchTrades(networkId, order)
     // Depending on order UID to avoid re-fetching when obj changes but ID remains the same
+    // Depending on `executedBuy/SellAmount`s string to force a refetch when there are new trades
+    // using the string version because hooks are bad at detecting Object changes
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [fetchTrades, networkId, order?.uid])
+  }, [fetchTrades, networkId, order?.uid, order?.executedSellAmount.toString(), order?.executedBuyAmount.toString()])
```

Comment:
Of course this is a hack ;)

Hope the comment is clear enough.
Can provide further details if needed.
But in that case I should also update the comment.

Reply:
This shouldn't really be the case should it? I feel we'd have seen this in other places too. But if you say it works..

Reply:
yikes! but fair enough, and with the comments sounds alright.